### PR TITLE
Tabs - Add possibility to display one tab

### PR DESCRIPTION
**Pull request by janosio-nordcloud**

### CODE REVIEW

File: src/components/tabs/Tabs.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ interface TabsProps {
   wizard?: boolean;
   name?: string;
   caption?: string;
-  children: Array<TabProps>;
+  children: [TabProps];
```
Comment (nowyDEV) [inline src/components/tabs/Tabs.tsx:30]: Are you sure it should be typed as a tuple here?

**Comment by raczyk** [comment]
It would be nice to align them a little better. Tabs should fulfil wrapper, or when there are two of them should have the width for 50% of the wrapper.

**Comment by janosio-nordcloud** [comment]
> It would be nice to align them a little better. Tabs should fulfil wrapper, or when there are two of them should have the width for 50% of the wrapper.

This task is the bugfix, needed in overwatch to show one tab properly.

**Review by raczyk — DISMISSED** [review]

File: src/components/tabs/Tabs.tsx [hunk 2]
```diff
@@ -219,34 +219,37 @@ export const Tabs: FunctionComponent<TabsProps> = ({
   handleTab,
   step,
 }) => {
+  const items = React.Children.toArray(children) as [TabProps];
```
Comment (nowyDEV) [inline src/components/tabs/Tabs.tsx:222]: as tuple?

**Review by nowyDEV — APPROVED** [review]

**Review by nowyDEV — APPROVED** [review]

**Review by raczyk — APPROVED** [review]

**Pull request merged**